### DISCUSSION TITLE
Interacting with objects while sitting on a chair

### DISCUSSION THREAD

**Discussion by AbyssBeast09** [FakeFishGames/Barotrauma#7568] [Ideas and suggestions]
In my opinion need to add ability interacting with items (Terminals, Fabricators, Cabinets and etc) while sitting on chair
In this moment we cannot interact with any type of objects while sitting

**Comment by MaaiG** [comment]
[Two years later](https://github.com/Regalis11/Barotrauma/issues/3031#issue-635571865).

**Reply by StruggleDuck** [reply]
your link is broken

**Reply by MaaiG** [reply]
> your link is broken

Now working. ))

**Comment by ekqvist** [comment]
Just wanted to let you all know that we're currently working on introducing interacting with devices while climbing ladders or sitting on chairs. Hopefully, we'll get to introduce it to Unstable soon™!

**Reply by MaaiG** [reply]
Thank you for informing! It’s nice to know that things are moving!
Now I will just be patient.
